src/Components/TopTab.jsx

```diff
@@ -20,10 +20,16 @@ const TopTab = () => {
   };
 
   return (
-    <div className="h-28 w-full relative">
+    <div
+      className="h-28 w-full"
+      style={{
+        backgroundImage: `url(${darkMode ? WaveDark : WaveLight})`,
```

Comment:
It looks like the Curve SVG is narrower now: I preferred how it was before. What do you think:

Before
![image](https://github.com/monkin77/cern-indico-check-in/assets/28673635/868bdeac-c6c8-47ba-86ef-22633bed7ee9)

After
![image](https://github.com/monkin77/cern-indico-check-in/assets/28673635/bc7e6547-71a9-4c4a-97cd-90b474975d8b)